TSPL.docc/LanguageGuide/Macros.md

```diff
@@ -505,9 +527,6 @@ dependencies: [
 ],
 ```
 
-Replace the `some-tag` placeholder in the code above
-with the Git tag for the version of SwiftSyntax you want to use.
```

Comment:
The example no longer has a placeholder.  I fixed this on 'main' after you last merged that in here, but then I must have picked up the deletion here while backing out content that was lost by that merge.

Reply:
Gotcha; I think I may have merged incorrectly, but I thought I had noticed an inconsistency where both the reference to "some tag" and the raw version number were included and I guess I resolved something wrong.

This makes sense now.